pkg/api/network.go

```diff
@@ -31,9 +32,37 @@ func (t *TitondAPI) GetNetworksByPage(page int) ([]model.Network, error) {
 	return networks, err
 }
 
-func (t *TitondAPI) GetNetworkByID(networkID uint) (*model.Network, error) {
+func (t *TitondAPI) GetNetworkByID(networkID uint) (interface{}, error) {
 	fmt.Println("Query Network by ID:", networkID)
-	return t.db.ReadNetwork(networkID)
+	network, err := t.db.ReadNetwork(networkID)
+	if err != nil {
+		return nil, err
+	}
+	deploymentMap := make(map[string]interface{})
+	if network.ContractAddressURL == "" {
+		deployment, err := t.GetK8sJobStatus(network)
+		fmt.Println("Update deployer status", deployment.Status, err)
+		if err == nil {
+			jsonData, err := json.MarshalIndent(deployment, "", "  ")
+			if err != nil {
+				deploymentMap["Error"] = err.Error()
+			} else {
+				err = json.Unmarshal(jsonData, &deploymentMap)
+				if err != nil {
+					deploymentMap["Error"] = err.Error()
+				}
+			}
+		} else {
+			deploymentMap["Error"] = "Could not find the deployment"
+		}
+		var data map[string]interface{}
+		data, err = ConvertStructToMap(*network)
+		if err == nil {
+			data["Message"] = deploymentMap
```

Comment:
Why you receive deployer's deployment to Message?

Reply:
It is actually status of deployment in object. So the message let us know about the deployment process

Reply:
In this case, the deployer's work is not yet finished (contractAddressURL == ""), are you checking the status of deployer's deployment?

Reply:
Yes, i am checking the status of deployer when deployer's job is not finished

Reply:
Thank you for your opinion

In actual, when testing the app, i need to access to K8s cluster to get info when it takes long time but not done, and it is quite inconvenient. 

I think it would be good if we provide it in query network so that we can know the issue if it happens.

Reply:
Thank you, Steven

I considered about that but it is quite difficult to be confident about the status of K8s 's job programmatically. It is easy when we look at the status of deployment and make guess. So I decided to response the raw status of deployment and user can make guess or even check the log in K8s's pods or .

Reply:
Thank you for your suggestion.
I will check on it.

Reply:
Thank you very much @ohbyeongmin

Reply:
Thank you guys for the suggestions!
I updated the code